# DO NOT CHANGE IT
Version: V4

# Choose what components are enabled on startup
SonarEnabled: false
TactilesEnabled: false
BumpersEnabled: false
FootContactsEnabled: false

# Camera related parameters
UseCamera: true
TopCameraEnabled: true
BottomCameraEnabled: true

# Application related parameters
PublishIMU: false

# ROS related paramaters
TopicQueue: 10

# Robot-Connection related parameters
RobotPort: 9559
RobotIP: 198.18.0.1
#RobotIP: 10.0.128.81
DriverBrokerPort: 54000
DriverBrokerIP: 0.0.0.0
CameraBrokerPort: 54001
CameraBrokerIP: 0.0.0.0

# Communication related parameters
HighCommunicationFrequency: 15
LowCommunicationFrequency: 10
# 100Hz are perfect for Webots, 5-10Hz works really beautiful on my real Nao
ControllerFrequency: 10
CameraFrequency: 15

# Robot related parameters
JointPrecision: 0.00174532925
OdomFrame: odom

# DO NOT CHANGE IT
#Prefix: romeo_dcm
